### PR TITLE
IAMU - Language-Independent IAC Binary Data

### PR DESCRIPTION
**Pull request by christianguenter2**
#81 Language-Independent Internet Application Components Binary Data

### CODE REVIEW

File: src/zabapgit_object_iamu.prog.abap [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+    TRY.
+        cl_w3_api_mime=>s_check_exist(
+          EXPORTING
+            p_mime_name = ls_mime_name
+          IMPORTING
+            p_exists    = rv_bool ).
+
+      CATCH zcx_abapgit_exception.
+        rv_bool = abap_false.
+        RETURN.
```
Comment (larshp) [inline src/zabapgit_object_iamu.prog.abap]: this RETURN is not needed, better just to have the method return normally

Comment (flaiker) [reply src/zabapgit_object_iamu.prog.abap]: Can `cl_w3_api_mime=>s_check_exist` even raise `zcx_abapgit_exception`? The try-catch doesn't make sense to me.

Comment (larshp) [reply src/zabapgit_object_iamu.prog.abap]: yeah, really need to finish https://github.com/larshp/abapOpenChecks/issues/109 sometime

Comment (christianguenter2) [reply src/zabapgit_object_iamu.prog.abap]: You're right the try catch block doesn't make any sense. It's an relict from a previous implementation. I'll remove it.

**Comment by christianguenter2** [comment]
Unnecessary try catch block is removed. Please review again.

**Pull request merged**